Source/Core/Core/CoreTiming.cpp

```diff
@@ -148,6 +151,18 @@ void Shutdown()
   UnregisterAllEvents();
 }
 
+void RefreshConfig()
+{
+  if (!s_has_registered_config_callback)
+  {
+    Config::AddConfigChangedCallback([]() { Core::RunAsCPUThread([]() { RefreshConfig(); }); });
+    s_has_registered_config_callback = true;
+  }
+
+  s_config_OC_factor =
+      Config::Get(Config::MAIN_OVERCLOCK_ENABLE) ? Config::Get(Config::MAIN_OVERCLOCK) : 1.0f;
```

Comment:
Perhaps consider also having an `s_config_OC_factor_inverted` to let us skip a division in `Advance`. (Maybe not all that important, but hey, Advance does run fairly often...)